pkg/store/mockstore/unistore/tikv/write.go

```diff
@@ -166,6 +167,11 @@ func (w writeLockWorker) run() {
 					// Ignore if the key doesn't exist
 					ls.DeleteWithHint(entry.Key.UserKey, hint)
 				default:
+					// Make sure it fits into an arena block.
+					if len(entry.Key.UserKey)+len(entry.Value) > ls.MaxEntrySize() {
+						batch.err = fmt.Errorf("Unistore lock entry too big %d > %d", len(entry.Key.UserKey)+len(entry.Value), ls.MaxEntrySize())
```

Comment:
```suggestion
						batch.err = fmt.Errorf("unistore lock entry too big %d > %d", len(entry.Key.UserKey)+len(entry.Value), ls.MaxEntrySize())
```

Errors in go shouldn't start with a capital right?

Reply:
https://go.dev/wiki/CodeReviewComments#error-strings (maybe not the best source of information) mentions this. But for Unistore this is fine either way I guess.